js/server/modules/@arangodb/aql-graph-traversal-generic-graphs.js

```diff
@@ -52,26 +52,31 @@ const TestVariants = Object.freeze({
 const graphWeightAttribute = 'distance';
 
 class TestGraph {
-  constructor(graphName, edges, eRel, vn, en, protoSmartSharding, testVariant, numberOfShards) {
+  constructor(graphName, edges, eRel, vn, en, protoSmartSharding, testVariant, numberOfShards, additionalVertices) {
```

Comment:
i do not get the change with the additionalVertices.

This constructor code creates empty collections as orphans.

In the test later the additionalVertices are used as source/target vertex, but not as collections.